src/parser/__main__.py

```diff
@@ -71,6 +71,13 @@
         else:
             case_data = post2017.parse(case_soup, case_id)
 
+        #Adds a hash to the JSON file of the underlying HTML
+        body = case_soup.find("body")
+        balance_table = body.find_all("table")[-1]
+        if "Balance Due" in balance_table.text:
```

Comment:
Might be good to leave a comment explaining why we're doing this

(In this case, the "balance due" table will change frequently even if the case data itself isn't updated, so it makes the hashes look unique even when things haven't changed. This code removes that table)

Reply:
Added comment. Thanks!